### ISSUE TITLE
Porting edgar to OS X

### ISSUE THREAD

**Issue by munepi** [riksweeney/edgar#2]
For the latest version, I uploaded the dmg image to the following URL
http://junk.greencherry.jp/macosx/edgar/edgar-1.21-Yosemite+-20150708.dmg{,.sha1sum}{,.sig}

The app contained in the dmg image works on OS X Yosemite 10.10 or higher version.

Best, 
Munehiro

**Comment by riksweeney** [comment]
Thanks, I've added it to the binary downloads.

**Comment by munepi** [comment]
For the latest version, I put the dmg image to the following URL
http://junk.greencherry.jp/macosx/edgar/edgar-1.22-Yosemite+-20151113.dmg{,.sha1sum}{,.sig}

Best, 
Munehiro

**Comment by riksweeney** [comment]
Thanks, I've added it to the downloads.

**Comment by munepi** [comment]
Hi, Richard @riksweeney , 

I have put the dmg image of the current version of edgar to the following URL. 
http://junk.greencherry.jp/macosx/edgar/edgar-1.24-ElCapitan+-20160616.dmg{,.sha1sum}{,.sig}

The app contained in the dmg image works on OS X El Capitan 10.11 or higher version.

I bundled repaired SDL_image of an issue with a visual glitch on OS X El Capitan 
(See some issues of BTSs in details https://github.com/Homebrew/homebrew-python/issues/281,  https://trac.macports.org/ticket/37453, http://stackoverflow.com/questions/34757499/sdl-image-visual-glitch-on-os-x-el-capitan)

Best, 
Munehiro

**Comment by riksweeney** [comment]
Excellent, thank you! I've added it to list of binaries for 1.24.

**Comment by munepi** [comment]
Hi, Richard @riksweeney ,

I wish you, this project, and a happy Belated New Year!
The news that Edgar has been updated to 1.26 is so wonderful.

I have built the latest version of the Legend of Edgar and have uploaded it to the following URL.
http://junk.greencherry.jp/macosx/edgar/edgar-1.26-ElCapitan+-20170221.dmg{,.sha1sum}{,.sig}
The app contained in the dmg image works on OS X El Capitan 10.11 or higher version.

Best,
Munehiro
